runtime/src/cost_tracker.rs

```diff
@@ -82,71 +120,37 @@ impl CostTracker {
             return;
         }
 
-        let (costliest_account, costliest_account_cost) = self.find_costliest_account();
-
         datapoint_info!(
             "cost_tracker_stats",
             ("bank_slot", bank_slot as i64, i64),
             ("block_cost", self.block_cost as i64, i64),
             ("transaction_count", self.transaction_count as i64, i64),
-            (
-                "number_of_accounts",
-                self.cost_by_writable_accounts.len() as i64,
-                i64
-            ),
-            ("costliest_account", costliest_account.to_string(), String),
-            ("costliest_account_cost", costliest_account_cost as i64, i64),
```

Comment:
Do we want to add some equivalent for these  back?

Reply:
Probably

Reply:
`costliest_account_cost` can be replaced with `total_chained_cost`, so we know how block does in regarding account limit. Probably wouldn't have `costliest_account` anymore, and `number_of_accounts` can be skipped - not really being used anyway